kafka/0.5/Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN echo "$MD5HASH  /tmp/kafka.tgz" | md5sum -c - &&\
     rm -f /tmp/kafka.tgz
 
 COPY ./log4j.properties $KAFKA_HOME/config/log4j.properties
+RUN mkdir $KAFKA_HOME/config.template && mv $KAFKA_HOME/config/* $KAFKA_HOME/config.template
```

Comment:
I read "template" and assumed it was for configuration files that were templates. May be worth considering a different name. WDYT?